modules/ibi-installer-installation-config.adoc

```diff
@@ -34,7 +34,13 @@ If the seed image requires a separate private registry authentication, add the a
 |Specification|Type|Description
 |`shutdown`|`boolean`|Specifies if the host shuts down after the installation process completes. The default value is `false`.
 |`extraPartitionStart`|`string`|Specifies the start of the extra partition used for `/var/lib/containers`. The default value is `-40Gb`, which means that the partition will be exactly 40Gb in size and uses the space 40Gb from the end of the disk. If you specify a positive value, the partition will start at that position of the disk and extend to the end of the disk.
-|`extraPartitionLabel`|`string`|The label of the extra partition you use for `/var/lib/containers`. The default label is `varlibcontainers`.
+|`extraPartitionLabel`|`string`|The label of the extra partition you use for `/var/lib/containers`.
```

Comment:
I prefer to specify the default in the table, as with the rest of the parameters

Reply:
Done, thanks